versioned_docs/version-v1.1/volume/create-volume.md

```diff
@@ -37,4 +37,10 @@ Description: Create a volume from the Volume page.
 1. Select an existing `Image`.
 1. Configure the `Size` of the volume.
 
-![create-image-volume](/img/v1.1/volume/create-image-volume.png)
\ No newline at end of file
+![create-image-volume](/img/v1.1/volume/create-image-volume.png)
+
+## Known Issues
+
+### Volume created is not appearing in the volume list
+
+On the volume list page, users with 'project-member' permissions cannot see the created volumes.
```

Comment:
```suggestion
**Summary**: If you are using Harvester from Rancher, after creating a volume, **project_member** users cannot find the created volume on the **Volumes** page.
**Workaround**: You can refresh the **Volumes** page and the volume should display as expected.
```

Reply:
1. Consider adding the affected versions
2. Not sure if refreshing the page or applying the `harvester-v1.1.2-patch1` is the correct workaround. If applying the patch is the workaround, please add the instructions as well
3. Consider linking to the github issue page